ws2812/ws2812.go

```diff
@@ -12,15 +12,27 @@ import (
 
 var errUnknownClockSpeed = errors.New("ws2812: unknown CPU clock speed")
 
+type DeviceType uint8
+
+const (
+	WS2812 DeviceType = iota // RGB
+	SK6812                   // RGBA / RGBW
+)
```

Comment:
If we would rather opt for making an entirely separate SK6812 device and leverage the same calls that Deivce.WriteByte uses, I'm happy to do that too.